src/features/create-dao/steps/DAOSummaryField.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+
+import { TextStack } from '@zero-tech/zui/components';
+
+import styles from './DAOSummaryField.module.scss';
+import classNames from 'classnames';
+
+interface DAOSummaryFieldProps {
+	className?: string;
+	label: string;
+	value: string | number;
+}
+
+export const DAOSummaryField: FC<DAOSummaryFieldProps> = ({
+	className,
+	label,
+	value,
+}) => (
+	<TextStack
+		className={classNames(styles.Field, className)}
+		label={label}
+		primaryText={value.toString()}
+		secondaryText=""
```

Comment:
Ah I've seen this a couple of times in my work now too. I think we need to update `TextStack` in `zUI` to make `secondaryText` optional. Doesn't make sense to have to pass an empty string for the prop here. I'll make a task and follow up